src/components/Widget/components/Conversation/components/Messages/styles.scss

```diff
@@ -12,7 +12,7 @@
 .widget-embedded {
   .messages-container {
     height: 100vh;
-    max-height: 100%;
```

Comment:
I feel like it would be easier and less dependant on other elements to just do : 
`
height: 100%;
max-height: 100%;
`
I checked, it works everywhere

Reply:
Here's what I have. Chrome 74.0.3729.169 left. Firefox 60.7.0esr right.

![Screenshot](https://user-images.githubusercontent.com/759578/58719697-54a9db00-839e-11e9-81c5-2fbee61a359d.png)

Reply:
did you change height to 100**%** and not 100**vh** ?

Reply:
Yes, forgot to mention that the screenshot is for `height: 100%; max-height: 100%;`. All four possible combinations of 100%/100vh appear to render the same here.

Reply:
I have 60.7.0esr (64-bit) and it works for me 🤔You changed the property on `.widget-embedded .messages-container` ?